languages/go/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Usage: ./build.sh [--release]
+
+check_command() {
+  if ! command -v "$1" &>/dev/null; then
+    printf '%s\n' "$1 is required to build locally. Please install $2."
+    exit 1
+  fi
+}
+
+check_command cargo Rust
+check_command go Go
+check_command npm Node.js
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+GO_LIB_DIR="$REPO_ROOT/languages/go/internal/cinterface/lib"
+
+pushd "$REPO_ROOT" || exit 1
+printf '%s\n\n' "Cleaning old builds..."
+rm -f languages/go/example/example && rm -rf "$GO_LIB_DIR"
+
+printf '%s\n\n' "Building binaries..."
+
+if [ "$1" = "--release" ]; then
+  cargo build --release
+else
+  cargo build
+fi
+
+npm i && npm run schemas
```

Comment:
We never want to run `npm i`, and `ci` would re-install every dependency. I suggest presuming the environment is. already bootstrapped like our other build scripts.